### PR TITLE
Improve immutable ui style

### PR DESCRIPTION
**Pull request by jwangyangls**
Signed-off-by: Yogi_Wang <yawang@vmware.com>
image
image
image

### CODE REVIEW

File: src/portal/src/i18n/lang/tr-tr-lang.json [hunk 1]
```diff
@@ -1233,15 +1233,15 @@
...
         "ADD_RULE": "ADD RULE",
         "ADD_RULE_HELP_1": "Click the ADD RULE button to add a rule.",
         "EDIT": "Edit",
         "DISABLE": "Disable",
         "ENABLE": "Enable",
         "DELETE": "Delete",
-        "ADD_TITLE": "Add Immutable Rule",
-        "ADD_SUBTITLE": "Specify a immutable rule for this project. All immutable rules are independently calculated and each rule can be applied to a selected list of repositories.",
+        "ADD_TITLE": "Add Immutable Tag Rule",
+        "ADD_SUBTITLE": "Specify an immutable tag rule for this project. All immutable tag rules are independently calculated and each rule can be applied to a selected list of repositories.",
```
Comment (renmaosheng) [inline src/portal/src/i18n/lang/tr-tr-lang.json]: what does " All immutable tag rules are independently calculated " mean?

**Comment by AllForNothing** [comment]
@jwangyangls Please add "padding-left: 0" to the "ACTION" button

File: src/portal/src/i18n/lang/tr-tr-lang.json (hunk 1 above)
Comment (jwangyangls) [reply src/portal/src/i18n/lang/tr-tr-lang.json]: I have communicated with Pm and owner, and I am thinking about it together.

File: src/portal/src/i18n/lang/zh-cn-lang.json [hunk 2]
```diff
@@ -1230,15 +1230,15 @@
...
         "ADD_RULE": "添加新规则",
         "ADD_RULE_HELP_1": "点击添加按钮可添加规则。",
         "EDIT": "修改",
         "DISABLE": "禁用",
         "ENABLE": "启用",
         "DELETE": "删除",
-        "ADD_TITLE": "添加不变的规则",
-        "ADD_SUBTITLE": "为当前项目指定不改变规则。所有不改变的规则独立计算并且适用于所有符合条件的仓库。",
+        "ADD_TITLE": "添加不可变的规则",
+        "ADD_SUBTITLE": "为此项目指定不可变的规则。注意：所有不可变的规则都将首先被独立计算，合并后最终获得不可变规则的集合。",
```
Comment (wy65701436) [inline src/portal/src/i18n/lang/zh-cn-lang.json]: 为此项目指定不可变的Tag规则。注意：所有不可变的Tag规则都将首先被独立计算，合并后最终获得不可变规则的集合。

File: src/portal/src/i18n/lang/zh-cn-lang.json [hunk 3]
```diff
@@ -1230,15 +1230,15 @@
         "ACTION": "操作"
     },
     "IMMUTABLE_TAG": {
-        "IMMUTABLE_RULES": "不变的规则",
+        "IMMUTABLE_RULES": "不可变的规则",
```
Comment (wy65701436) [inline src/portal/src/i18n/lang/zh-cn-lang.json]: 不可变的Tag规则"

**Pull request merged**